pkg/skaffold/deploy/helm.go

```diff
@@ -186,109 +203,152 @@ func (h *HelmDeployer) helm(ctx context.Context, out io.Writer, useSecrets bool,
 	cmd.Stdout = out
 	cmd.Stderr = out
 
+	logrus.Infof("calling: %v", cmd.Args)
```

Comment:
this is already being logged (at debug level) in the command runner itself

Reply:
Removed.